packages/admin-theme/src/typographyOptions.ts

```diff
@@ -59,4 +59,11 @@ export const typographyOptions: TypographyOptions = {
         lineHeight: "20px",
         fontWeight: fontWeights.fontWeightRegular,
     },
+    button: {
+        textTransform: "inherit",
+        fontSize: "inherit",
+        lineHeight: "inherit",
+        fontWeight: "inherit",
+        color: "inherit",
+    },
```

Comment:
I don't think we need this part anymore. 

The only reason this exists is because in the beginning I incorrectly assumed, the text in every button needs to be wrapped with `<Typography variant="button" />`. I couldn't find any other instances of this variant in any of our projects, other than old buttons that I will refactor shortly.